packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -2135,11 +2136,12 @@ export class AiAgentModel {
                 versionUuid: version.ai_artifact_version_uuid,
                 title: version.title,
                 description: version.description,
-                chartConfig: version.chart_config,
-                dashboardConfig: version.dashboard_config,
+                chartConfig: version.chart_config as AiArtifact['chartConfig'],
+                dashboardConfig:
+                    version.dashboard_config as AiArtifact['dashboardConfig'],
```

Comment:
I don''t think we need to `toolSchemaArgs.parse` here, as we save `toolSchemaArgs.parse` output to db.

Transformed schema will take care of the backwards compat later on.

Ideally I would like to refactor this to do `toolSchemaArgsTransformed.parse` here and everything downstream uses transformed schemas